src/index.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import {render} from 'react-dom';
+import { BrowserRouter} from 'react-router-dom';
```

Comment:
Why `BrowserRouter` in this PR?. Router setup has already been assign to @gbols

Reply:
okay, will resolve asap